resources/migrations/000_migrations.yaml

```diff
@@ -3927,3 +3927,13 @@ databaseChangeLog:
                   type: text
                   constraints:
                     nullable: false
+  - changeSet:
+    - id: 68
+    - author: sbelak
+    - changes:
+      - addColumn:
+            tableName: computation_job
+            columns:
+              - column:
+                  name: context
```

Comment:
what goes in `context`? That's a suspiciously generic term and sounds like "stuff I'm going to be tempted to be too lazy to write a migration for"

Reply:
Ideally it's a map along the lines of what with-async does (see https://github.com/metabase/metabase/pull/6084/files#diff-379ea8cfdc1ba646068d26184b48c21fR107), but I left it ill-defined so that you are not forced to use the macro. My thinking was that sometimes you might want a different cache resolution, and having context flexible allows you that, but it may also be YAGNI. Along the same lines is it correct that caching and async are *so* coupled, or should caching be built on top of async but in such a way that async can still be used on it's own without resorting to tricks (you can for instance avoid caching by setting context to (gensym) or similar, but ...).